cmake/pcl_targets.cmake

```diff
@@ -905,3 +905,10 @@ endmacro()
 macro(PCL_SET_TEST_DEPENDENCIES _var _module)
   set(${_var} global_tests ${_module} ${PCL_SUBSYS_DEPS_${_module}})
 endmacro()
+
+macro (PCL_ADD_COMPILETIME_AND_RUNTIME_TEST _name _exename)
```

Comment:
Some comment about it adding `PCL_RUN_TESTS_AT_COMPILE_TIME` target could be added for documentation

Reply:
right, done